python/tvm/micro/testing/evaluation.py

```diff
@@ -32,20 +32,29 @@
 
 
 def tune_model(
-    platform, board, target, mod, params, num_trials, tuner_cls=tvm.autotvm.tuner.GATuner
+    platform,
+    board,
+    target,
+    mod,
+    params,
+    num_trials,
+    tuner_cls=tvm.autotvm.tuner.GATuner,
+    project_options=None,
 ):
     """Autotunes a model with microTVM and returns a StringIO with the tuning logs"""
     with tvm.transform.PassContext(opt_level=3, config={"tir.disable_vectorize": True}):
         tasks = tvm.autotvm.task.extract_from_program(mod["main"], {}, target)
     assert len(tasks) > 0
     assert isinstance(params, dict)
 
+    project_options = {
+        **(project_options if project_options is not None else {}),
```

Comment:
Two more nits about this change:
1. In `tune_model`, the `project_options` are inserted **above** where `project_type` is specified, meaning _`project_options` cannot be used to override the `project_type` parameter_. This differs from the implementation in `create_aot_session`, where they are inserted **below** and thus **can** override this parameter. We should one behavior and stick with it - I weakly prefer allowing us to override `project_type`.
2. This code can be rewritten as:
```suggestion
        **(project_options or {}),
```

Reply:
Thanks for the comments. I weakly preferred not allowing to overwrite the project type, but in the interest of sticking with one behavior, I followed the implementation in _create_aot_session_ .